tools/build-cocoa.sh

```diff
@@ -96,3 +96,59 @@ else
+            if [[ "$bt" == "Release" ]]; then
+                make_core_xcframework+=" -library temp/librealm-${p}${suffix}.a -headers core/include/"
+            else
+                make_core_debug_xcframework+=" -library temp/librealm-${p}${suffix}.a -headers core/include/"
+            fi
+        elif [[ "$p" == "ios" ]]; then
+            cp build-${p}-${bt}/src/realm/${bt}-iphoneos/librealm${suffix}.a temp/librealm-iphoneos${suffix}.a
+            cp build-${p}-${bt}/src/realm/${bt}-iphonesimulator/librealm${suffix}.a temp/librealm-iphonesimulator${suffix}.a
+            if [[ "$bt" == "Release" ]]; then
+                make_core_xcframework+=" -library temp/librealm-iphoneos${suffix}.a -headers core/include/"
+                make_core_xcframework+=" -library temp/librealm-iphonesimulator${suffix}.a -headers core/include/"
+            else
+                make_core_debug_xcframework+=" -library temp/librealm-iphoneos${suffix}.a -headers core/include/"
+                make_core_debug_xcframework+=" -library temp/librealm-iphonesimulator${suffix}.a -headers core/include/"
+            fi
+        elif [[ "$p" == "watchos" ]]; then
+            cp build-${p}-${bt}/src/realm/${bt}-watchos/librealm${suffix}.a temp/librealm-watchos${suffix}.a
+            cp build-${p}-${bt}/src/realm/${bt}-watchsimulator/librealm${suffix}.a temp/librealm-watchsimulator${suffix}.a
+            if [[ "$bt" == "Release" ]]; then
+                make_core_xcframework+=" -library temp/librealm-watchos${suffix}.a -headers core/include/"
+                make_core_xcframework+=" -library temp/librealm-watchsimulator${suffix}.a -headers core/include/"
+            else
+                make_core_debug_xcframework+=" -library temp/librealm-watchos${suffix}.a -headers core/include/"
+                make_core_debug_xcframework+=" -library temp/librealm-watchsimulator${suffix}.a -headers core/include/"
+            fi
+        elif [[ "$p" == "tvos" ]]; then
+            cp build-${p}-${bt}/src/realm/${bt}-appletvos/librealm${suffix}.a temp/librealm-appletvos${suffix}.a
+            cp build-${p}-${bt}/src/realm/${bt}-appletvsimulator/librealm${suffix}.a temp/librealm-appletvsimulator${suffix}.a
+            if [[ "$bt" == "Release" ]]; then
+                make_core_xcframework+=" -library temp/librealm-appletvos${suffix}.a -headers core/include/"
+                make_core_xcframework+=" -library temp/librealm-appletvsimulator${suffix}.a -headers core/include/"
+            else
+                make_core_debug_xcframework+=" -library temp/librealm-appletvos${suffix}.a -headers core/include/"
+                make_core_debug_xcframework+=" -library temp/librealm-appletvsimulator${suffix}.a -headers core/include/"
+            fi
+        fi
+    done
+done
+make_core_xcframework+=" -output core/xcframework/realm-core.xcframework"
+make_core_debug_xcframework+=" -output core/xcframework/realm-core-dbg.xcframework"
+
+eval $make_core_xcframework
```

Comment:
Rather than using `eval` you could use a bash array to build up the arguments. it'd look something like `make_core_xcframework=()`, `make_core_xcframework+=( -library temp/librealm-appletvos${suffix}.a -headers core/include/)` inside the loop, and `xcodebuild -create-xcframework "${make_core_xcframework[@]}"` to invoke it.